### PR TITLE
Notify Slack channel when meeting videos added to youtube

### PR DESCRIPTION
**Pull request by patcon**
This will run whenever the script find a video to upload. It will include a link to the playlist in which the video is now available.

### CODE REVIEW

**Comment by patcon** [comment]
- [x] set `SLACK_WEBHOOK_URL` in jenkins job after merge using value from [here](https://edgi.slack.com/services/237846976401?updated=1)

File: .env.sample [hunk 1]
```diff
@@ -1,2 +1,3 @@
 EDGI_ZOOM_API_KEY=
 EDGI_ZOOM_API_SECRET=
+NOTIFICATION_WEBHOOK_URL=https://my-hubot.herokuapp.com/youtube-upload-complete
```
Comment (Mr0grog) [inline .env.sample:3]: Given the example URL, this variable name seems overly generic and prone to confusion.

File: scripts/upload_zoom_recordings.py [hunk 2]
```diff
@@ -61,8 +62,10 @@ def download_file(url, download_path):
 
         print('Recording is permitted for upload!')
         for file in meeting['recording_files']:
+            uploaded_files = 0
             if file['file_size'] == 0:
                 print('Meeting still processing: {}'.format(meeting['topic']))
+                requests.post(NOTIFICATION_WEBHOOK_URL, {'status':'processing'})
```
Comment (Mr0grog) [inline scripts/upload_zoom_recordings.py:68]: This raises an exception if you pass `False` (the default) for the URL. Should it be guarded by an `if` statement?

**Comment by patcon** [comment]
indirect :+1: from @trinberg [in Slack](https://edgi.slack.com/archives/C3EF5T7T6/p1517241561000038) on this being an improvement

**Pull request closed without merging**